src/lib/pickles/wrap_domains.ml

```diff
@@ -28,7 +28,11 @@ struct
     let dummy_step_keys =
       lazy
         (Vector.init num_choices ~f:(fun _ ->
-             let g = Backend.Tock.Inner_curve.(to_affine_exn one) in
+             let num_chunks = (* TODO *) 1 in
```

Comment:
We should use `Vector` to encode the number of chunks, and change the type `array` accordingly :). Chunking safety at the type level :).

Reply:
Or use a type alias instead of `array` just for readability, like `type chunks = array`. We would have `Affine.t chunks` or smth like this. IMHO, it reminds that the type is related to a change in chunking.

Reply:
We should, but I would much rather get the cryptography working first. Feel free to attempt that change though.

Reply:
Opened follow-up issue https://github.com/MinaProtocol/mina/issues/14315